data-reconciliation-app/docs/adr/01-data-ingest.md

```diff
@@ -0,0 +1,139 @@
+# Data Ingest API
+
+## Status
+
+Proposed
+
+## Context
+
+We need to build an API that ingests member data into our system. Once ingested, we also need to store this data in the K-V store.
+
+Gathered requirements:
+
+- Members will upload data via csv
+- Members will upload multiple records (batch) or single records via csv
+- Data will have one attribute, and we will only work with string attributes for now.
+
+## Decision
+
+### Endpoint
+
+```
+POST  /csv
+// Endpoint ingests data into data reconciliation app
+	• URL Params
+        None
+	• Headers
+        Content-Type: application/json
+	• Request is ccfapp.Request object
+		○ Reuest Body = CSV text
+		○ Request UserID =
+	• Status: 200
+    • Status Text: "Data has been successfully recorded"
+```
+
+### Service Layer - CSV Service
+
+```
+// injects member data into CCF K-V store
+SubmitData(userId, csv body))
+	• Parse spreadsheet
+		○ Any validation error
+        ○ Grab headers of spreadsheet
+		○ Check if key exists
+            - if exists, update vote
+            - else, save to K-V Store
+```
+
+### K-V Store data structure
+
+- Key: Unique Id (string)
+- Value: Attribute object
+  - id: attribute header (string)
```

Comment:
This `id` is confusing when we have `unique id` in the Key.
Is this actually `companyName`?